tests/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { PrismaClient } from "@trigger.dev/database";
+
+type SetDBCallback = (prisma: PrismaClient) => Promise<void>;
+
+export const setDB = async (cb: SetDBCallback) => {
+  const { DATABASE_URL } = process.env;
+
+  const prisma = new PrismaClient({
+    datasources: {
+      db: {
+        url: DATABASE_URL,
+      },
+    },
+  });
+
+  try {
```

Comment:
remove the try .. if the connection not success the test should be fail ... ASAP..

Reply:
Removed the `try/catch`